lib/r10k/api.rb

```diff
@@ -88,72 +91,78 @@ def parse_puppetfile(io_or_content)
         parser.instance_eval(io_or_content)
       end
 
-      return { :modules => builder.build }
+      return builder.build
```

Comment:
This caused me to go look at EnvmapBuilder, and I'll admit that it bothers me that EnvmapBuilder.build does *not* build an Envmap. It builds an array of modules. That really feels like it needs to be named .build_modules.